spec/swift/app_spec.rb

```diff
@@ -1,827 +1,56 @@
 require_relative "../spec_helper"
 
 describe "App" do
```

Comment:
Same as above, a more meaningful description like "Swift provider" would be better.